tests/letstest/testdata/sample-config/accounts/acme-staging.api.letsencrypt.org/directory/48d6b9e8d767eccf7e4d877d6ffa81e3/meta.json

```diff
@@ -0,0 +1 @@
+{"creation_host": "ec2-52-91-193-99.compute-1.amazonaws.com", "creation_dt": "2016-12-23T02:08:32Z"}
```

Comment:
This is a test file generated by Certbot! If we want it to end with a newline, we should change our code :)